source/_docs/automation/trigger.markdown

```diff
@@ -198,6 +198,36 @@ automation:
 
 {% endraw %}
 
+When the `attribute` option is specified the trigger is compared to the given `attribute` instead of the state of the entity.
+
+{% raw %}
+
+```yaml
+automation:
+  trigger:
+    - platform: numeric_state
+      entity_id: climate.kitchen
+      attribute: current_temperature
+      above: 23
+```
+
+{% endraw %}
+
+More dynamic and complex calculations can be done with `value_template`.
+
+{% raw %}
+
+```yaml
+automation:
+  trigger:
+    - platform: numeric_state
+      entity_id: climate.kitchen
+      value_template: "{{ state.attributes.current_temperature - state.attributes.temperature_set_point }}"
+      above: 3
+```
+
+{% endraw %}
+
```

Comment:
This should be removed.

---

PS: Never use `state.attributes.x` in your automation or templates.

![image](https://user-images.githubusercontent.com/195327/127904601-fa676d14-f10a-4981-9cfa-40f557bcb1a1.png)

Reply:
Is `state` as bad as `states`? 🤔 I'm far from an expert here, so please correct me if I misunderstand anything. Wouldn't the `state` variable always be available in this specific case? Because it's bound to the entity in the trigger.

Presuming it is bad, what's the alternative? Is `state_attr` even available in this context? Reading https://www.home-assistant.io/docs/configuration/templating/#limited-templates hints that it might not be.

Reply:
This is not a limited template.